source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -482,6 +485,15 @@ driver MUST start in Unknown topology, and follow the rules in the
 the driver MUST NOT use the number of hosts from the initial SRV lookup
 to decide what topology to start in.
 
+A default connection string option for ``loadBalanced=true`` MUST be valid
```

Comment:
Both this paragraph and the next are duplicated from the LB spec. Can they be removed from here?

Reply:
Removed.